src/components/common/ColonyActions/ActionDetailsPage/DefaultMotion/MotionPhaseWidget/StakingWidget/helpers.ts

```diff
@@ -111,3 +117,11 @@ export const getUserStakeLimitPercentage = (
     minUserStake,
   );
 };
+
+export const getUserStakes = (
+  usersStakes: MotionData['usersStakes'],
```

Comment:
You define this as the GraphQL type, which isn't necessarily what you query for (e.g. `UserStake` has `address` and `stakes` field, but your query might not ask the backend for both of those fields). It's safer to declare a GraphQL fragment and use the generated type for that